client/src/components/Home/Home.js

```diff
@@ -0,0 +1,15 @@
+import React, { Component } from 'react';
+import { Typography } from '@material-ui/core';
+
+class Home extends Component {
```

Comment:
Can we start favoring functional components as opposed to class based components?